tapa/verilog/xilinx/module.py

```diff
@@ -227,6 +227,12 @@ def ports(self) -> dict[str, IOPort]:
             if isinstance(x, Input | Output | Inout)
         )
 
+    class NoMatchingPortError(ValueError):
+        """No matching port being found exception."""
+
+        def __init__(self, message: str) -> None:
```

Comment:
You can just put a `pass` here, e.g., https://github.com/UCLA-VAST/soda/blob/720e4e2ddb143a468e1f2d0283e47e4056c4d3db/src/soda/model/resource.py#L193